apps/atlas/src/hooks/settling/useRealmsPlaylist.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable sonarjs/cognitive-complexity */
+
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import toast from 'react-hot-toast';
+import type { RealmWhereInput } from '@/generated/graphql';
+import { storage } from '@/util/localStorage';
+import useKeyPress from '../useKeyPress';
+import usePrevious from '../usePrevious';
+
+export const realmPlaylistCursorKey = 'realm.playlist.cursor';
+export const realmPlaylistNameKey = 'realm.playlist.name';
+export const realmPlaylistKey = 'realm.playlist';
+
+export type PlaylistQueryType = 'AllRealms' | 'Account' | 'Ids' | 'Raidable';
+type GetPlaylistQueryFilterFunction = (args?: any) => RealmWhereInput;
+
+export const playlistQueryStrategy: Record<
+  PlaylistQueryType,
+  GetPlaylistQueryFilterFunction
+> = {
+  AllRealms: () => ({}),
+  Account: (account: string) => ({ settledOwner: { equals: account } }),
+  Ids: (ids: number[]) => ({ realmId: { in: ids } }),
+  Raidable: () => ({
+    NOT: [
+      {
+        lastVaultTime: { equals: null },
+      },
+    ],
+  }),
+};
+
+export const resetPlaylistState = () => {
+  storage(realmPlaylistCursorKey, 0).remove();
```

Comment:
![Screen Shot 2022-08-16 at 6 06 18 PM](https://user-images.githubusercontent.com/91395140/185012349-8c4545c9-9814-4396-b59d-672a8bb6d168.png)
got this error on load

Reply:
went away after reload. can't reproduce

Reply:
have not seen this before. It's clear that remove is a function that exists, maybe this was a 1-time anomaly having to do with hot module reloading or something.